package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "lint": "eslint frontend/src/**/*js frontend/cypress/**.js && stylelint frontend/src/**/*.scss && npm run puglint",
-    "puglint": "pug-lint frontend/src/index.pug frontend/src/ramp.pug frontend/src/summary.pug frontend/src/tabs/authorship.pug frontend/src/tabs/segment.pug frontend/src/tabs/zoom.pug",
+    "puglint": "pug-lint frontend/src/index.pug frontend/src/ramp.pug frontend/src/summary.pug frontend/src/summary_charts.pug frontend/src/resizer.pug frontend/src/tabs/authorship.pug frontend/src/tabs/segment.pug frontend/src/tabs/zoom.pug",
```

Comment:
An unrelated thing (create a new issue if need be): Is it possible to just blanket it with `pug-lint frontend/src/**.pug`, like what's done with the others?

So we don't have to fiddle with this everytime we create/remove a .pug file

Reply:
Hmm its an intended fix from #1136 actually; We could go back to `frontend/src/**.pug` once #795 is done ( `pug-lint-vue` expands globs unlike `pug-lint` )

`frontend/src/summary_charts.pug` is unrelated though, I didn't add it in before #1136 was merged. Should this be in a separate pr? 😮

Reply:
Hmm its an intended fix from #1136 actually; We could go back to `frontend/src/**.pug` once #795 is done ( `pug-lint-vue` expands globs unlike `pug-lint` )

`frontend/src/summary_charts.pug` is unrelated though, I didn't add it in before #1136 was merged. Should this be in a separate pr? 😮

Reply:
I see. We can leave it for now.